charts/4allportal/templates/webdav/configmap.yaml

```diff
@@ -124,4 +127,63 @@ data:
     BrowserMatch " Konqueror/4" redirect-carefully
     BrowserMatch "^gvfs" redirect-carefully
     BrowserMatch "WebDAVLib" redirect-carefully
+  log-handler.sh: |
+    #!/bin/ash
+    ENDPOINT="http://{{ include "common.names.fullname" . }}-backend/api/extensions/dam/fs-event"
+    while IFS= read -r line; do
+      status=$(echo "$line" | cut -d';' -f1)
+      type=$(echo "$line" | cut -d';' -f2)
+      path=$(echo "$line" | cut -d';' -f3)
+      dest=$(echo "$line" | cut -d';' -f4 | awk '{
+        i = index($1,"/webdav/");
+        out = substr($1, i);
+        for (i = 0x20; i < 0x40; ++i) {
+          repl = sprintf("%c", i);
+          if ((repl == "&") || (repl == "\\")) {
+            repl = "\\" repl;
+          }
+          out = gensub(sprintf("%%%02X", i), repl, "g", out);
+          out = gensub(sprintf("%%%02x", i), repl, "g", out);
+        };
+        print out}')
+
+      matched=$(expr match "${path}" '/webdav.*')
+      if [ \( "${type}" == "PUT" -o "${type}" == "DELETE" -o "${type}" == "MKCOL" -o "${type}" == "MOVE" -o "${type}" == "COPY" \) -a ${matched} != 0 ]
+      then
+        OP="Unknown"
+        if [ "${status}" == "201" -a "${type}" == "PUT" ]
+        then
+          OP="CREATE"
+        elif [ "${status}" == "201" -a "${type}" == "MKCOL" ]
+        then
+          OP="CREATE"
+        elif [ "${status}" == "201" -a "${type}" == "MOVE" ]
+        then
+          OP="MOVE"
+        elif [ "${status}" == "201" -a "${type}" == "COPY" ]
+        then
+          OP="CREATE"
+        elif [ "${status}" == "204" -a "${type}" == "PUT" ]
+        then
+          OP="UPDATE"
+        elif [ "${status}" == "204" -a "${type}" == "DELETE" ]
+        then
+          OP="DELETE"
+        fi
+        BODY="{}"
+        if [ "${dest}" == "-" ]
+        then
+          BODY="{\"mount\":\"mount\",\"operation\":\"$OP\",\"path\":\"$path\"}"
+          echo "Send notification to $ENDPOINT: $BODY" > /proc/self/fd/1
+        else
+          BODY="{\"mount\":\"mount\",\"operation\":\"$OP\",\"path\":\"$path\",\"destination\":\"$dest\"}"
```

Comment:
The \"mount\":\"mount\" needs to be replaced with \"mount\":\"$mount\", where $mount variable is defined through a HelmValue.

Reply:
Done